pages/news/2020-XX-XX-new-in-2-3-serato-support.html

```diff
@@ -11,12 +11,14 @@
 
 As all of you probably know, Mixxx is the best DJ software in the market.
 However, we've recently been made aware that a minor competitor has emerged and tries to steal that title from us - [Serato DJ Pro](https://serato.com/dj/pro).
-Since Serato is neither open-source nor as customizable, we're not too concerned.
+The hardware support is nice, but since Serato is neither open-source nor as customizable, we're not too concerned.
 From the user's perpective, switching to the most popular open-source DJ software in world seems like the obvious choice.
 
 Unfortunately, migrating from Serato to Mixxx is a lot of work - apart from familiarizing yourself with a new tool, all the countless hours you spent to organize and prepare the tracks in your old library are lost, and your need to start from scratch.
 At least until now - if you're a Serato user and cast and eye on Mixxx, we've got a nice surprise for you.
```

Comment:
cast **an** eye

Reply:
I like the above paragraph very mucho :)